readthedocs/search/tests/test_views.py

```diff
@@ -312,7 +312,7 @@ def test_file_search_filter_by_project(self, client):
             search_params=search_params,
         )
         project_facets = facets['project']
-        resulted_project_facets = [facet[0] for facet in project_facets]
+        resulted_project_facets = [ facet[0] for facet in project_facets ]
```

Comment:
questa spaziatura la possiamo lasciare com'è in originale?

Reply:
Spazi presenti nel codice RTD (vedi Nota)